code/script.js

```diff
@@ -0,0 +1,79 @@
+// DOM selectors
+
+// HTML sections to inject into
+const main = document.getElementById("projects");
+const userInfo = document.getElementById("user-info");
+
+const username = "Smelbows";
+const USERS_REPOS_API = `https://api.github.com/users/${username}/repos`;
+
+const fetchUserRepos = (userUrl) => {
+  fetch(userUrl)
+    .then((res) => res.json())
+    .then(filterForTechnigoRepos)
+    .then((repos) => {
+      userInfo.innerHTML = createHTMLForUser(repos[0]);
+      repos.forEach((repo) => {
+        main.innerHTML += createHTMLForRepo(repo);
+        fetchPullRequests(repo);
+      });
+      drawChart(repos);
+    });
+};
+
+const filterForTechnigoRepos = (data) => {
+  return data.filter((repo) => repo.name.startsWith("project"));
+};
+
+const createHTMLForUser = (repo) => {
+  return `<img class="avatar" src=${repo.owner.avatar_url} alt="github avatar"/><p class="name">${repo.owner.login}</p>`;
+};
+
+const createHTMLForRepo = (repo) => {
+  return `<a target="blank" href="${repo.html_url}" class="repo-container ${
+    repo.language
```

Comment:
Can I ask what does this <repo. language> do in the code? Maybe I am missing something that is actually quite obvious, but was curious what this code means.